src/goose/synopsis/toolkit.py

```diff
@@ -67,33 +71,16 @@ def shell(self, command: str) -> str:
         self.logshell(command)
         return shell(command, self.notifier, self.exchange_view, cwd=system.cwd, env=system.env)
 
-    @tool
-    def read_file(self, path: str) -> str:
-        """Read the content of the file at path
-
-        Args:
-            path (str): The destination file path, in the format "path/to/file.txt"
-        """
-        system.remember_file(path)
-        self.logshell(f"cat {path}")
-        return f"The file content at {path} has been updated above."
-
-    @tool
-    def write_file(self, path: str, content: str) -> str:
-        """
-        Write a file at the specified path with the provided content. This will create any directories if they do not exist.
-        The content will fully overwrite the existing file.
-
-        Args:
-            path (str): The destination file path, in the format "path/to/file.txt"
-            content (str): The raw file content.
-        """  # noqa: E501
+    def _write_file(self, path: str, content: str) -> str:
+        """Write content to the file at path."""
         patho = system.to_patho(path)
 
         if patho.exists() and not system.is_active(path):
             print(f"We are warning the LLM to view before write in write_file, with path={path} and patho={str(patho)}")
             raise ValueError(f"You must view {path} using read_file before you overwrite it")
 
+        self._save_file_history(patho)  # Save current content to history
```

Comment:
:nit: i see a lot of erroneous comments like these in this pr and most of the time they're already self self documenting. maybe remove?